bindings/pyroot_experimental/PyROOT/src/PyROOTModule.cxx

```diff
@@ -31,6 +31,8 @@ static PyMethodDef gPyROOTMethods[] = {{(char *)"AddBranchAttrSyntax", (PyCFunct
                                         (char *)"Allow to access branches as tree attributes"},
                                        {(char *)"SetBranchAddressPyz", (PyCFunction)PyROOT::SetBranchAddressPyz, METH_VARARGS,
                                         (char *)"Pythonization for TTree::SetBranchAddress"},
+                                       {(char *)"BranchPyz", (PyCFunction)PyROOT::BranchPyz, METH_VARARGS,
```

Comment:
The comment is fine but perhaps we can choose something which is less generic than "Pythonization for TTree::Branch?"